config/locales/en.yml

```diff
@@ -13,13 +13,14 @@ en:
       footer_manage: "Manage your email preferences"
     digests:
       daily:
-        subject: "Daily update from GOV.UK"
-        opening_line: Daily update from GOV.UK.
-        permission_reminder: "^You’re getting this email because you subscribed to daily updates on these topics on GOV.UK."
+        subject: "Daily update from GOV.UK for: %{title}"
+        heading: "Yesterday’s changes"
```

Comment:
Should we use curly quotes here? Designers seem to love them, so I'm wondering if they should be in here :-)

(I think that's a larger conversation, so let's not do that now)

Reply:
Oh yes, they're intentional here: we've done similarly for confirmation emails ([here](https://github.com/alphagov/email-alert-api/blob/55726c8f73aabda0fd58588a6d3a942d3603ee86/config/locales/en.yml#L5-L7)).